osu.Game.Tests/Visual/Gameplay/TestSceneBeatmapSkinFallbacks.cs

```diff
@@ -84,18 +84,18 @@ protected bool AssertComponentsFromExpectedSource(SkinnableTarget target, ISkin
             Remove(expectedComponentsAdjustmentContainer);
 
             return almostEqual(actualInfo, expectedInfo);
-
-            static bool almostEqual(SkinnableInfo info, SkinnableInfo other) =>
-                other != null
-                && info.Type == other.Type
-                && info.Anchor == other.Anchor
-                && info.Origin == other.Origin
-                && Precision.AlmostEquals(info.Position, other.Position)
-                && Precision.AlmostEquals(info.Scale, other.Scale)
-                && Precision.AlmostEquals(info.Rotation, other.Rotation)
-                && info.Children.SequenceEqual(other.Children, new FuncEqualityComparer<SkinnableInfo>(almostEqual));
         }
 
+        private static bool almostEqual(SkinnableInfo info, SkinnableInfo other) =>
```

Comment:
Is this failure somehow related to the changes in this PR? o.O

Reply:
I haven't investigated too deeply, so I'm open for help here, but I believe it may be related to offsetting position by height of score by default?

Reply:
@smoogipoo's hunch is partially right, it is related to the offsetting. The rest of it is that the offsetting logic [is using `ScreenSpaceDrawQuad.Size.Y`](https://github.com/ppy/osu/blob/4e3d9da22d4422f2eb3b3e1f0c513ac95af45d4a/osu.Game/Skinning/DefaultSkin.cs#L87) rather than just plain `Height`. I'm not actually super sure why that would be required - I'd like to think all HUD skinnables should be in a common coordinate space.